make/modules/java.base/Launcher.gmk

```diff
@@ -78,7 +78,11 @@ ifeq ($(call isTargetOsType, unix), true)
       NAME := jspawnhelper, \
       SRC := $(TOPDIR)/src/$(MODULE)/unix/native/jspawnhelper, \
       OPTIMIZATION := LOW, \
-      CFLAGS := $(CFLAGS_JDKEXE) -I$(TOPDIR)/src/$(MODULE)/unix/native/libjava, \
+      CFLAGS := $(CFLAGS_JDKEXE) -I$(TOPDIR)/src/$(MODULE)/unix/native/libjava \
+          -DVERSION_FEATURE=$(VERSION_FEATURE) \
+          -DVERSION_INTERIM=$(VERSION_INTERIM) \
+          -DVERSION_UPDATE=$(VERSION_UPDATE) \
+          -DVERSION_PATCH=$(VERSION_PATCH), \
```

Comment:
Using all 4 is way overkill for the problem at hand.  Just the FEATURE_VERSION is sufficient.
We all know better than to make incompatible changes in minor versions let alone update or patch version.

Reply:
There is already a `$(VERSION_CFLAGS)` variable defined. It will set all those (and some more). Please use it instead. But then, as Roger says, it is probably overkill to *check* anything but the feature version.